boards/FSM/Src/App/App_FlowMeter.c

```diff
@@ -24,7 +24,12 @@ struct FlowMeter *App_FlowMeter_Create(float (*get_flow_rate)(void))
     return flow_meter;
 }
 
-float App_FlowMeter_ReadFlowRate(struct FlowMeter *flow_meter)
+float App_FlowMeter_ReadFlowRate(const struct FlowMeter *const flow_meter)
```

Comment:
Thought I left this earlier. Consistent naming for getter functions
```suggestion
float App_FlowMeter_GetFlowRate(const struct FlowMeter *const flow_meter)
```